<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>371</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="chartPic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>881</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Demo_Project_Gantt.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="scheduledButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Scheduled Commits</string>
   </property>
  </widget>
  <widget class="QPushButton" name="actualButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>290</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Actual Commits</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Scheduled Gantt Chart</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
